src/odemis/dataio/test/tiff_test.py

```diff
@@ -1289,7 +1289,7 @@ def testExportSmallPyramid(self):
         self.assertEqual(subimage[-1][0], 10022)
         self.assertEqual(subimage[-1][-1], 10278)
 
-        del im
```

Comment:
Why changing the `del` into a `.close()`? Aren't they equivalent? (and `del` is nicer as it's the standard behaviour of Python)